centreon/www/class/config-generate/generate.class.php

```diff
@@ -78,6 +79,7 @@
 require_once __DIR__ . '/timezone.class.php';
 require_once __DIR__ . '/vault.class.php';
 require_once __DIR__ . '/AdditionalConnectorVmWareV6.class.php';
+require_once __DIR__ . '/AgentConfiguration.class.php';
 
```

Comment:
I don't understand, it's useless because there is an autoload of the directory /class, have you done a `composer dump-autoload` before the test ?

Reply:
Seen together, we'll leave it like that in case of potential autoload problems